mars/oscar/backends/pool.py

```diff
@@ -869,6 +869,8 @@ async def handle_control_command(self,
                 processor.result = ResultMessage(message.message_id, True,
                                                  protocol=message.protocol)
             elif message.control_message_type == ControlMessageType.wait_pool_recovered:
+                while not await self.is_sub_pool_alive(self.sub_processes[message.address]):
+                    await asyncio.sleep(.5)
```

Comment:
The `wait_pool_recovered` behavior is different from before. @wjsi Is it OK?

Reply:
I wonder if it is possible to create an event and wait for it. Repeative querying is not a good practice.

Reply:
> I wonder if it is possible to create an event and wait for it. Repeative querying is not a good practice.

Yes, there is a `_recover_events` and it's better to reuse it. In this case we should deal with it more carefully.